ui/apps/platform/cypress/integration/compliance/hideScanResults.helpers.js

```diff
@@ -0,0 +1,52 @@
+import { interactAndWaitForResponses } from '../../helpers/request';
+
+import { routeMatcherMapWithoutStandards } from './Compliance.helpers';
+
+// selectors
+
+export const selectorForModal = `[role="dialog"]:has('h1:contains("Manage standards")')`;
```

Comment:
Related question:
Is <h1> the latest recommendation for the main dialog heading?

Reply:
Good question. It has 2 interpretations:
* Is it the recommended markup? PatternFly `Modal` element renders given `title="Manage standards"` prop.
* Is it the recommended selector for tests? Use of `role` attribute and `h1` element is inspired by philosophy of testing-library and, I think, our general philosophy for cypress to assert what users experience.